Gems/PhysX/Code/physx_files.cmake

```diff
@@ -104,6 +104,7 @@ set(FILES
     Include/PhysX/Debug/PhysXDebugConfiguration.h
     Include/PhysX/Debug/PhysXDebugInterface.h
     Include/PhysX/Configuration/PhysXConfiguration.h
+    Include/PhysX/Joint/JointRequestsBus.h
```

Comment:
I would actually keep the filename as it was `PhysXJointBus.h` for 2 reasons:
- This header can potentially include more buses related to Joints in the future, it's quite common that a include is called `FeatureBus.h` and the inside it has `FeatureRequestsBus` and `FeatureNotificationBus`.
- All the other joint files have the PhysX prefix, so for consistency I'd keep it.

Reply:
Ok